manifests/d/dremin/RetroBar/1.15/dremin.RetroBar.installer.yaml

```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: dremin.RetroBar
+PackageVersion: "1.15"
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: RetroBar.exe
+  PortableCommandAlias: retrobar
+Installers:

```

Comment:
```suggestion
Dependencies:
  PackageDependencies:
     - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.6
Installers:
```

Reply:
With the dependency, this package appears to install and run normally: 
![image](https://github.com/microsoft/winget-pkgs/assets/114952565/02e9cece-9cab-4eae-8808-1fecd2b13f9a)